.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  publish:
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
Query: Is this a default setting for travis? Looking at the original travis config it was just executed on every change no? If this is changing the behavior please adjust the README.

Reply:
It was, but then they checked the tag on the script. This was moved out in favor of removing TRAVIS_TAG

Reply:
Make sense! 👍🏻